sdk/python/kfp/compiler/_component_builder.py

```diff
@@ -455,7 +432,13 @@ def build_docker_image(staging_gcs_path, target_image, dockerfile_path, timeout=
     namespace (str): the namespace within which to run the kubernetes kaniko job, default is "kubeflow"
   """
   _configure_logger(logging.getLogger())
-  builder = ComponentBuilder(gcs_staging=staging_gcs_path, target_image=target_image, namespace=namespace)
-  image_name_with_digest = builder.build_image_from_dockerfile(docker_filename=dockerfile_path, timeout=timeout)
+
+  with tempfile.TemporaryDirectory() as local_build_dir:
+    dst_dockerfile_path = os.path.join(local_build_dir, 'Dockerfile')
```

Comment:
These two 'Dockerfile' are related to each other, might as well declare one variable to keep them consistent.

Reply:
Makes sense.
Done.